x/programs/rust/examples/pokemon/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "pokemon"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+expose_macro = { version = "0.1.0", path = "../../expose_macro" }
+wasmlanche_sdk = { version = "0.1.0", path = "../../wasmlanche_sdk" }
+rand = "0.8.5"
+rand_chacha = "0.3.1"
+serde_derive = "1.0.185"
+serde = "1.0.185"
+serde_json = "1.0.105"
```

Comment:
If the goal for performance is to minimize the footprint required to manage the stack I am wondering if this is the right move to take these dependencies. Why do we need JSON?